mobilelib.php

```diff
@@ -156,6 +156,32 @@ function build_allowed_tag_users($userarray) {
     return $returnarray;
 }
 
+/**
+ * @param context_course $context
+ * @param $user
+ * @return bool
+ */
+function check_capability($context, $user)
+{
+    global $result, $USER;
+
+    try {
+        if (has_capability('local/getsmarter:mention_' . $user->shortname, $context)) {
+            return true;
+        }
+    } catch (Exception $e) {
+        error_log($e);
+        $result->result = false;
+        $result->content = $e;
+
+        header('Content-type: application/json');
+        echo json_encode($result);
+        die();
```

Comment:
Would termination cause any issues down the line? Is it an AJAX call?

Reply:
Good call - this was code that was used from web. Believe we can remove the die()

Reply:
Good point, think this was copied from something I wrote and might be leftover debugging code.